wikidataQuery.py

```diff
@@ -45,3 +81,24 @@ def submitQuery(entity_id, property_id):
         for var in item :
             answers.append(item[var]['value'])
     return answers
+    
+# Creates a query that checks whether the given property
+# has appropriate attribute and returns the yes/no answer on that query.
+def submitCheckQuery(entity_id, property_id, attribute_id):
+    url = 'https://query.wikidata.org/sparql'
+    query = '''
+        ASK {{
+            wd:{0} wdt:{1} ?attribute .
+            FILTER(?attribute = wd:{2})
+        }}
+        '''.format(entity_id, property_id, attribute_id)
+
+    data = requests.get(url, params={'query': query, 'format': 'json'}).json()
+
+    answer = []
+    if data['boolean'] == True:
+        answer = ['Yes']
+    else:
+        answer = ['No']
+    
+return answer
```

Comment:
Did you not get errors while running this file? Because the last return statement is outside the function body.